test/RouterFactoryTest.php

```diff
@@ -17,10 +17,13 @@
 use Laminas\ServiceManager\Config;
 use Laminas\ServiceManager\ServiceManager;
 use PHPUnit\Framework\TestCase;
+use Prophecy\PhpUnit\ProphecyTrait;
 
 class RouterFactoryTest extends TestCase
 {
-    public function setUp()
+    use ProphecyTrait;
```

Comment:
Can you recheck the entire test class if the Prophecy is really needed here? Thanks in advance!